PROTO_tests/tests/index_test.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+import arkouda as ak
+
+
+class TestIndex:
+    def test_index_creation(self):
+        idx = ak.Index(ak.arange(5))
```

Comment:
It looks like we could use pytest.prob_size for many of these.

Reply:
Updated, also added some ternaries for @pierce314159 :)